gamestonk_terminal/portfolio/README.md

```diff
@@ -60,4 +70,39 @@ usage: rhhist [-s --span] [-i --interval]
     
 Example Default Output:
 
-![rhhist](https://user-images.githubusercontent.com/18151143/111718919-36da8e00-8831-11eb-99e1-957c8eccb583.png)
\ No newline at end of file
+![rhhist](https://user-images.githubusercontent.com/18151143/111718919-36da8e00-8831-11eb-99e1-957c8eccb583.png)
+## ALPACA <a name="ALPACA"></a>
+
+Alpaca has a nicely maintained python API that is used here.  To login, the best approach is to just save the following environment variables:
+* "APCA_API_BASE_URL"
```

Comment:
To follow others API Keys, probably use something like: GT_ALP_BASE/KEY/SECRET

Reply:
So I was going to keep it like that, but the alpaca api actually goes in and looks for those specific env variables.  Anyone using Alpaca probably has those stored as well.  

I can change it to keep the same convention if you would like.

Also - yes reqs are updated (from yesterday)

Reply:
On that basis, I agree with you to leave it as is! Feel free to merge it